src/Symfony/Bundle/WebServerBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.4.0
+-----
+
+ * 'WebServer can now use '*' as a wildcard to bind to 0.0.0.0 (INADDR_ANY)
```

Comment:
you have an extra quote at the beginning. and the value should use backticks to make it a Markdown code span (especially given that `*` is a special char otherwise)